frontend/src/components/main-nav.js

```diff
@@ -134,6 +140,13 @@ class MainNav extends React.Component {
       background-color: var(--toast-purple-transparent);
       mix-blend-mode: screen;
     }
+    .link-active{
```

Comment:
i would change this to the following css:
background-color: var(--toast-purple-2);
mix-blend-mode: luminosity;

you can just call this class .icon-caption:active when you switch back to the a tag